src/Returns/Http/Responses/ReturnResponse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Wms\Returns\Http\Responses;
+use Illuminate\Contracts\Support\Responsable;
+use Illuminate\Http\JsonResponse;
+readonly class ReturnResponse implements Responsable
+{
+    /**
+     * @param string $wmsReturnId The unique identifier assigned to the return by the Warehouse Management System (WMS).
+     */
+    public function __construct(
+        private string $wmsReturnId
+    ) {
+    }
```

Comment:
Readonly classes are a bit tricky when extending them. Lets keep this particular class not readonly so that the wms integrations get the extensibility to extend it.

```suggestion
use Illuminate\Http\JsonResponse;

class ReturnResponse implements Responsable
{
    /**
     * @param string $wmsReturnId The unique identifier assigned to the return by the Warehouse Management System (WMS).
     */
    public function __construct(
        private readonly string $wmsReturnId
    ) {
    }
```